test/instrumentation/decode_metric.go

```diff
@@ -182,7 +183,24 @@ func (c *metricDecoder) decodeOpts(expr ast.Expr) (metric, error) {
 				if err != nil {
 					return m, err
 				}
+			case *ast.SelectorExpr:
+				packageName := fmt.Sprintf("%v", v.X)
```

Comment:
hacky, there's a better way to do this below in this file.  I will update later.

Reply:
performing an explicit cast instead of SprintF for fame